lib/cisco_node_utils/cmd_ref/interface_portchannel.yaml

```diff
@@ -57,6 +57,15 @@ lacp_suspend_individual:
     auto_default: false
     default_value: true
 
+lacp_suspend_shut_needed:
+  # This is used to determine if the port-channel interface
+  # must be shutdown before setting lacp_suspend_individual
+  kind: boolean
+  N3k:
```

Comment:
this problem is happening on n9k only as of now, so only shut down the interface for n9k. let us not change other platform behavior since they may or may not change.